background.ts

```diff
@@ -415,14 +416,8 @@ Electron.ipcMain.on('k8s-restart', async() => {
   try {
     switch (k8smanager.state) {
     case K8s.State.STOPPED:
-      await k8smanager.start();
-      break;
     case K8s.State.STARTED:
-      await k8smanager.stop();
-      // The desired Kubernetes version might have changed
-      k8smanager = newK8sManager(cfg.kubernetes);
-
-      await k8smanager.start();
+      await k8smanager.start(cfg.kubernetes);
```

Comment:
Could use a comment that calling `k8smanager.start` when it's running will switch versions. At the surface a reader would wonder why call `.start()` when it's running.